src/lib/containers/access_requirements/HasAccessV2.tsx

```diff
@@ -165,14 +163,8 @@ export function useGetFileHandleDownloadType(
 }
 
 /**
- * HasAccess shows if the user has access to the file or not.
- *
- * The component's behavior changes whether it's passed in a FileHandle or not.
- * If passed a file handle then it will give more detailed information about the download.
- *
- * @export
- * @class HasAccess
- * @extends {React.Component<HasAccessProps, HasAccessState>}
```

Comment:
Outdated comment from HasAccess v1, referencing functionality that was removed long ago